spec/requests/topics_controller_spec.rb

```diff
@@ -2512,6 +2512,13 @@ def extract_post_stream
       expect(response.media_type).to eq('application/rss+xml')
     end
 
+    it 'renders rss even if post is deleted' do
+      topic.posts.map(&:destroy)
```

Comment:
The PR description says "soft deleted" (meaning `.trash!`?) but the spec here is performing `.destroy`. Should one of them be changed?